### DISCUSSION TITLE
Mark / color / rewrite title on tabs for ordering/grouping?

### DISCUSSION THREAD

**Discussion by kekscode** [qutebrowser/qutebrowser#6909] [Ideas] [answered]
Quick question before i open a feature request or something similar: I would think it is useful to be able to rewrite the text of a tab or color/mark it in some way. The reason: I sometimes have lots of documentation open in several tabs (can be up to 100). I'd like to be able to mark a tab with e.g. a color like: "all blue tabs are Kubernetes docs" and "all red tabs are redhat openshift docs" and "all green tabs are news articles i want to read in my spare time" or something similar. I don't see that this is possible right now, or am I wrong?

Anybody else thinks that this would be useful? I could also serve as a base for a feature like: `:group-tabs <color>` or something like that which would groups tabs by color and maybe by the time it had been opened or something like that (to make it intuitive).

**Comment by The-Compiler** [comment, marked as answer]
See:

[Tree style tabs · Issue #927 · qutebrowser/qutebrowser](https://github.com/qutebrowser/qutebrowser/issues/927)
[tab groups / tagging of tabs · Issue #49 · qutebrowser/qutebrowser](https://github.com/qutebrowser/qutebrowser/issues/49)
